src/h3lib/lib/faceijk.c

```diff
@@ -28,12 +28,13 @@
 #include "coordijk.h"
 #include "geoCoord.h"
 #include "h3Index.h"
+#include "vec3d.h"
 
 /** square root of 7 */
 #define M_SQRT7 2.6457513110645905905016157536392604257102L
 
 /** @brief icosahedron face centers in lat/lon radians */
-static const GeoCoord faceCenterGeo[NUM_ICOSA_FACES] = {
+const GeoCoord faceCenterGeo[NUM_ICOSA_FACES] = {
```

Comment:
For my info: Why drop `static`?

Reply:
Static is incompatible with extern, because a static variable would only be visible within the same compilation unit (think .c file) and extern is needed for the generator.

Reply:
You don't need `extern` here if this is declared `extern` in a header file, which is the case here. I'd remove the `extern` here.

Reply:
My mistake. Looks good.